powergenome/nrelatb.py

```diff
@@ -1083,7 +1086,7 @@ def atb_new_generators(atb_costs, atb_hr, settings):
     new_gen_df = new_gen_df[keep_cols]
     # Set no capacity limit on new resources that aren't renewables.
     new_gen_df["Max_Cap_MW"] = -1
-
+    new_gen_df["Max_Cap_MWh"] = -1
```

Comment:
Good catch. I didn't realize that I automatically set Max_Cap_MW to -1. Makes sense to do the same for Max_Cap_MWh.